src/tiles/list/tile-list-compact.ts

```diff
@@ -66,9 +68,14 @@ export class TileListCompact extends LitElement {
   private get href(): string {
     // Use the server-specified href if available.
     // Otherwise, construct a details page URL from the item identifier.
-    return this.model?.href
-      ? `${this.baseNavigationUrl}${this.model.href}`
-      : `${this.baseNavigationUrl}/details/${this.model?.identifier}`;
+    if (this.model?.href) {
+      return `${this.baseNavigationUrl}${this.model?.href}`;
+    }
+
+    const isCollection = this.model?.mediatype === 'collection';
+    return `${this.baseNavigationUrl}${
+      isCollection ? this.collectionPagePath : '/details/'
+    }${this.model?.identifier}`;
```

Comment:
QUESTION: Does this optional chain value need a protection against being undefined? e.g. `?? ''`?

Reply:
Yes -- in fact we can probably just have an early-out at the top of the method to ensure the model is defined. If it's not, there's little point in trying to render the link anyway.